broker/thinBroker.go

```diff
@@ -2495,16 +2536,35 @@ func (tb *ThinBroker) LDUpdateEntityAttributes(w rest.ResponseWriter, r *rest.Re
 				} else {
 					err := tb.updateAttributes(deSerializedAttributePayload, eid)
 					if err != nil {
-						rest.Error(w, err.Error(), 207)
+						rest.Error(w, err.Error(), 404)
 						return
 					}
 					w.WriteHeader(204)
 				}
 			}
 		} else {
-			ERROR.Println("The entity was not found!")
-			rest.Error(w, "The entity was not found!", 404)
-			return
+			tb.ldEntities_lock.RUnlock()
+			ownerURL := tb.queryOwnerOfLDEntity(eid)
+			if ownerURL != tb.MyURL {
+				ownerURL = strings.TrimSuffix(ownerURL, "/ngsi10")
+				reqCxt, _ := tb.getStringInterfaceMap(r)
+				//link := r.Header.Get("Link") // Pick link header if present
+				//fmt.Println("Here 1..., link sending to remote broker:", link, "\nOwner URL:", ownerURL, "\nMy URL:", tb.MyURL)
+				_, code := tb.updateLDAttributeValues2RemoteSite(reqCxt, ownerURL, eid)
+				if code == 207 {
+					//rest.Error(w, err.Error(), 404)
+					//ERROR.Println(err)
```

Comment:
[minor] it should have proper comment.
Better to consider 207 status code as error because its partial success code/error/success status code.